skiff-mail-web/hooks/useQuickActions.tsx

```diff
@@ -89,7 +96,12 @@ export const useQuickActions = (query: string): Array<SearchAction> => {
       { icon: Icon.Folder },
       undefined // no shortcut
     );
-
+    const toggleThreadFormatAction = createAction(
+      'Toggle full/split view',
```

Comment:
I think we should change the text to dynamically update
```suggestion
      `Toggle to ${threadFormat === ThreadDisplayFormat.Full ? 'split' : 'full'} view`,
```

Reply:
good suggestion, consider it done

Reply:
or even `Switch to ${threadFormat === ThreadDisplayFormat.Full ? 'split' : 'full'} view` if we're changing text dynamically since "toggle" feels more apt when both options are presented

Reply:
Implemented in https://github.com/marcoleder/skiff-apps/commit/b64a2d0207b4bf8ef21d9a9df85e1cf6f7602b9f